### PR TITLE
Update Entry: General IDE

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From Target Keyword List.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/ide/ide.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-An IDE, or Integrated Development Environment, enables programmers to consolidate the different aspects of writing a computer program.
+An **IDE**, or Integrated Development Environment, enables programmers to consolidate the different aspects of writing a computer program.
 
 IDEs increase programmer productivity by combining common activities of writing software into a single application: editing source code, building executables, and debugging.
```
Comment (Dusch4593) [inline content/general/concepts/ide/ide.md:21]: For the header on the next line, I would suggest renaming it to "Common Features". Then, I would add a hashtag `#` to the following headers:

- ~Autocomplete~ (change this to "Autocompletion")
- Building Executables
- Debugging

File: content/general/concepts/ide/ide.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-An IDE, or Integrated Development Environment, enables programmers to consolidate the different aspects of writing a computer program.
+An **IDE**, or Integrated Development Environment, enables programmers to consolidate the different aspects of writing a computer program.
```
Comment (Dusch4593) [inline content/general/concepts/ide/ide.md:18]:
```suggestion
An **Integrated Development Environment (IDE)**, enables programmers to consolidate the different aspects of writing a computer program.
```

File: content/general/concepts/ide/ide.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ CatalogContent:
...
 
 IDEs increase programmer productivity by combining common activities of writing software into a single application: editing source code, building executables, and debugging.
```
Comment (Dusch4593) [inline content/general/concepts/ide/ide.md:20]:
```suggestion
IDEs increase programmer productivity by featuring common activities of writing software into a single application..
```

File: content/general/concepts/ide/ide.md [hunk 4]
```diff
@@ -85,3 +85,11 @@ There are numerous IDEs available, such as:
 - [Microsoft Visual Studio](https://visualstudio.microsoft.com)
 
 The IDE you decide to use depends on what area of computer programming you wish to pursue. Some IDEs, like Android Studio and Xcode, are dedicated solely for building mobile applications.
+
+### Online IDEs
+
+There are many online IDEs that allow users to practice with snippets of code without installing any software on their local machine. Most provide support for several different languages.
+
+One example is Codecademy's own [Codebyte Editor](https://www.codecademy.com/codebyte-editor). It allows experimenting with [C++](https://www.codecademy.com/resources/docs/cpp), C#, [Go](https://www.codecademy.com/resources/docs/go), [JavaScript](https://www.codecademy.com/resources/docs/javascript), [PHP](https://www.codecademy.com/resources/docs/php), [Python 3](https://www.codecademy.com/resources/docs/python), [Ruby](https://www.codecademy.com/resources/docs/ruby) and Scheme.
```
Comment (Dusch4593) [inline content/general/concepts/ide/ide.md:93]:
```suggestion
One example is Codecademy's own [Codebyte Editor](https://www.codecademy.com/codebyte-editor). It allows experimenting with [C++](https://www.codecademy.com/resources/docs/cpp), C#, [Go](https://www.codecademy.com/resources/docs/go), [JavaScript](https://www.codecademy.com/resources/docs/javascript), [PHP](https://www.codecademy.com/resources/docs/php), [Python 3](https://www.codecademy.com/resources/docs/python), [Ruby](https://www.codecademy.com/resources/docs/ruby), and Scheme.
```

File: content/general/concepts/ide/ide.md [hunk 5]
```diff
@@ -85,3 +85,11 @@ There are numerous IDEs available, such as:
...
+
+Another example is [JDoodle](https://www.jdoodle.com/) which offers support for over 75 languages including [Java](https://www.codecademy.com/resources/docs/java), [Python 2](https://www.codecademy.com/resources/docs/python), [R](https://www.codecademy.com/resources/docs/r) and [Swift](https://www.codecademy.com/resources/docs/swift).
```
Comment (Dusch4593) [inline content/general/concepts/ide/ide.md:95]:
```suggestion
Another example is [JDoodle](https://www.jdoodle.com/) which offers support for over 75 languages including [Java](https://www.codecademy.com/resources/docs/java), [Python 2](https://www.codecademy.com/resources/docs/python), [R](https://www.codecademy.com/resources/docs/r), and [Swift](https://www.codecademy.com/resources/docs/swift).
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review and left some comments/suggestions.

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestions

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! 👍🏻 I think we can just merge this with one review since this is only edits for one file.

**Pull request merged**